rpcserver.go

```diff
@@ -3874,14 +3874,12 @@ func (r *rpcServer) fetchWaitingCloseChannels(
 		if closingTx != nil {
 			closingTxid = closingTx.TxHash().String()
 			if includeRawTx {
-				var txBuf bytes.Buffer
-				err = closingTx.Serialize(&txBuf)
+				closingTxHex, err = lnrpc.SerializeAndHexEncodeTx(
```

Comment:
The linter complains about the line being too long. Maybe the function should instead be called `SerializeTx` and return bytes, then the hex encoding can be done here still (then it would be a more useful function since in most places we actually send raw bytes and not hex encoded transactions). I would've suggested to do the same here, but it looks like the other RPC that includes the raw transaction also uses a `string` instead of `bytes`. So I guess consistency is slightly better than wire-level efficiency in this case.